dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -2399,6 +2400,7 @@ DomainParticipantImpl::LivelinessTimer::handle_timeout(
   const ACE_Time_Value& tv,
   const void* /* arg */)
 {
+  Thread_Monitor::Green_Light gl("DomainParticipant");
```

Comment:
Does the name passed to the constructor need to be unique (thinking of the case where there are two or more DomainParticipants)?